src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -1937,13 +1937,9 @@
 	<!-- Default style for Windows.UI.Xaml.Controls.TimePicker -->
 	<!--https://msdn.microsoft.com/en-us/library/windows/apps/mt299156.aspx-->
 	<xamarin:Style TargetType="TimePicker">
-		<Setter Property="IsTabStop"
-				Value="False" />
-		<Setter Property="MinWidth"
-				Value="242" />
-		<Setter Property="MaxWidth"
-				Value="456" />
-		<Setter Property="FontFamily"
+        <Setter Property="IsTabStop"
```

Comment:
Is it for the new native style? I don't remember. What happens with the default XAML style from MS's generic.xaml?